lib/surface/catalogue/example.ex

```diff
@@ -41,13 +41,13 @@ defmodule Surface.Catalogue.Example do
       @config unquote(opts)
       @before_compile unquote(__MODULE__)
 
-      import Surface, except: [sigil_H: 2]
+      import Surface, except: [sigil_F: 2]
 
-      defmacrop sigil_H({:<<>>, meta, [string]} = ast, opts) do
+      defmacrop sigil_F({:<<>>, meta, [string]} = ast, opts) do
```

Comment:
```suggestion
      defmacrop sigil_F({:<<>>, _meta, [string]} = ast, opts) do
```

it doesn't look like we actually use this variable :-)

Reply:
I wonder why there's no warning 🤔

Reply:
could be because it's quoted? though i'd potentially expect warnings on `use Surface.Catalogue.Example` if that were the case